tables/he-common-vowels-ihbc.uti

```diff
@@ -0,0 +1,476 @@
+always שֲׁ 146-25  # shin with dot and hataf patah
+always שֲּׁ 5-146-25  # shin with dot and hataf patah
+always שֲׂ 156-25  # sin with hataf patah
+always שֲּׂ 5-156-25  # sin  hataf patah
+always שֳּׁ 5-146-345  # shin  hataf qamats
+always שֳּׂ 5-156-345  # sin  hataf qamats
+always שֳׁ 146-345  # shin with dot and hataf qamats
+always שֳּׁ 5-146-345  # shin with dot and hataf qamats
+always שֳׂ 156-345  # sin with hataf qamats
+always שֳּׂ 5-156-345  # sin  hataf qamats
+always שִּׁ 5-146-24  # shin with hiriq
+always שִּׂ 5-156-24  # sin with hiriq
+always שִׁ 146-24  # shin  with dot and hiriq
+always שִּׁ 5-146-24  # shin  with dot and hiriq
+always שִׂ 156-24  # sin with hiriq
+always שִּׂ 5-156-24  # sin with hiriq
+always שֵּׁ 5-146-34  # shin with tsere
+always שֵּׂ 5-156-34  # sin with tsere
+always שֵׁ 146-34  # shin with dot and tsere
+always שֵּׁ 5-146-34  # shin with dot and tsere
+always שֵׂ 156-34  # sin with tsere
+always שֵּׂ 5-156-34  # sin with tsere
+always שֶּׁ 5-146-15  # shin with segol
+always שֶּׂ 5-156-15  # sin with segol
+always שֶׁ 146-15  # shin with dot and segol
+always שֶּׁ 5-146-15  # shin with dot and segol
+always שֶׂ 156-15  # sin and segol
+always שֶּׂ 5-156-15  # sin with segol
+always שַּׁ 5-146-14  # shin with patah
+always שַּׂ 5-156-14  # sin with patah
+always שַׁ 146-14  # shin with dot and patah
+always שַּׁ 5-146-14  # shin with dot and patah
+always שַׂ 156-14  # sin and patah
+always שַּׂ 5-156-14  # sin with patah
+always שָּׁ 5-146-126  # shin  qamats
+always שָּׂ 5-156-126  # sin  qamats
+always שָׁ 146-126  # shin with dot and qamats
+always שָּׁ 5-146-126  # shin with dot and qamats
+always שָׂ 156-126  # sin with qamats
+always שָּׂ 5-156-126  # sin  qamats
+always שְּׁ 5-146-3  # Shin with dot dagesh and sheva
+always שְּׂ 5-156-3  #  Sin with dagesh and sheva
+always שְׁ 146-3  #  Shin with dot and sheva
+always תְ 1256-3  # tav with dagesh sheva
+always תֱּ 1256-26  # tav with dagesh and hataf segol
+always תֲּ 1256-25  # tav with dagesh and hataf patah
+always תֳּ 1256-345  # tav with dagesh and hataf qamats
+always תִּ 1256-24  # tav  with dagesh and hiriq
+always תֵּ 1256-34  # tav with dagesh and tsere
+always תֶּ 1256-15  # tav with dagesh and segol
+always תַּ 1256-14  # tav with dagesh and patah
+always תָּ 1256-126  # tav with dagesh and qamats
+always תְּ 1256-3  # tav with dagesh sheva
+always תֱּ 1256-26  # tav with dagesh and hataf segol
+always תֲּ 1256-25  # tav with dagesh and hataf patah
+always תֳּ 1256-345  # tav with dagesh and hataf qamats
+always תִּ 1256-24  # tav  with dagesh and hiriq
+always תֵּ 1256-34  # tav with dagesh and tsere
+always תֶּ 1256-15  # tav with dagesh and segol
+always תַּ 1256-14  # tav with dagesh and patah
+always תָּ 1256-126  # tav with dagesh and qamats
+always בְּ 12-3  # Bet with dagesh sheva
+always כְּ 13-3  # KAF with dagesh  sheva
+always תְּ 1256-3  # tav with daggesh sheva
+
+letter Bֳֳ 345  # hataf qamats
```

Comment:
This line is malformed and will likely break the table

Reply:
This was fixed.